demux/demux_textsub.c

```diff
@@ -0,0 +1,159 @@
+/*
+ * This file is part of mpv.
+ *
+ * mpv is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * mpv is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with mpv.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <limits.h>
+#include <math.h>
+
+#include <subrandr/subrandr.h>
+
+#include "common/common.h"
+#include "demux/packet.h"
+#include "misc/bstr.h"
+#include "options/m_config.h"
+#include "options/m_option.h"
+#include "stream/stream.h"
+#include "demux.h"
+
+#define OPT_BASE_STRUCT struct demux_textsub_opts
+struct demux_textsub_opts {
+    int probesize;
+};
+
+const struct m_sub_options demux_textsub_conf = {
+    .opts = (const m_option_t[]) {
+        {"demuxer-textsub-probesize", OPT_INT(probesize), M_RANGE(32, INT_MAX)},
```

Comment:
```suggestion
        {"probesize", OPT_INT(probesize), M_RANGE(32, INT_MAX)},
```
move prefix to upper level.